Sources/Kitura/CodableRouter.swift

```diff
@@ -425,7 +425,20 @@ public struct CodableHelpers {
         }
         return contentType.hasPrefix("application/json")
     }
-
+    
+    /**
+     * Check if the given request has content type x-www-form-urlencoded
+     *
+     * - Parameter request: The RouterRequest to check
+     * - Returns: True if the content type of the request is application/x-www-form-urlencoded, false otherwise
+     */
+    public static func isContentTypeURLEncoded(_ request: RouterRequest) -> Bool {
+        guard let contentType = request.headers["Content-Type"] else {
+            return false
+        }
+        return contentType.hasPrefix("application/x-www-form-urlencoded")
```

Comment:
This is executed twice on the critical path (by `read()`, and by `readCodableOrSetResponseStatus()` which itself calls `read()`).  Each execution involves a dictionary lookup and a call to `String.hasPrefix()`, both of which involve String comparison.

It would be better to cache this state against the Request itself (eg. in an enum), particularly if we end up supporting more content types in the future.  For now though, it's a trivial extra cost and we can make a note to fix it in the future.